### DISCUSSION TITLE
#50 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Emilyxiannv** [tog-ETH-er/togETHer#32] [General]
Agenda:

-Community updates
-Issue discussion:
https://github.com/tog-ETH-er/tog-eth-er.github.io/issues/10
-The sixth rotating chairman nomination
-Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by bambalados** [comment]
**Crypto Tech Night**

- Done：英文 Twitter Space 关于 SBT 与 vitae3 & TampabayDAO - 10.24号
- Planning: 做市商与安全性 - 10.03号

**Comment by stephanieeee-ho** [comment]
### ECN
1. 常规的翻译和周报输出
2. 与 imToken 的研究员 Nic Lin 录制了一期关于 4844 的播客，估计这周五会上线 https://ecnpodcast.fireside.fm/

**Comment by luckyyang** [comment]
### Rebase
- web3 daily 正常的每日更新 https://rebase.network/posts/category/web3daily
- 招聘: 5 个团队发布多个职位 https://github.com/rebase-network/who-is-hiring/issues
- 活动: 支持 2 个活动，自己组织了分享《Rebase大学｜BNBChain垮链攻击事件深度分析》 https://github.com/rebase-network/event-pool/issues
- 联合主办的 Hacker house 第一期结束，产出 11 个项目 https://www.notion.so/hackerhouses/Hacker-House-Dashboard-b0c07e2c076d488d8e439b8c62eb2867

**Comment by bobjiang** [comment]
**GitcoinCN**

- Gitcoin 中文周报两篇 #38 #39 - http://news.gitcoinchina.com/
- GR16 - Jan 2023
- budget for season 16

**Comment by Stamp9** [comment]
### 原语里弄

研究流程优化

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

8个PR #856 ～ #862

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《artgobblers-1: GOO & VRGDAs 设计原理 》 by mc
  - https://www.youtube.com/watch?v=qGWsPSAZGTc
  - https://www.youtube.com/watch?v=iaEQlh1CVFY

- 《artgobblers-2: 经济模型设计 》 by mc
  - https://www.youtube.com/watch?v=aRWOuKBeRKc

**Comment by 0x68616f79756e** [comment]
**ETHPlanet**
参与了COSCon 上海线下分会场的分享以及展示。

**Comment by homuncilus** [comment]
**PlanckerDAO**

- 运营小组
1. 成员 onboarding 流程优化
2. SoulWallet 分享（Rebase、Dapplearning、链茶馆）
3. 和北邮链协联合主办小 V 共读会

- 研究小组
1. ERC3525 分享
2. SBT 研究

- 建设小组
1. CSBS 项目 0.1 close，准备 0.2 的 kickoff
2. 建设小组完成 SOP 初稿和月度规划，帮助社区和人记录各种有趣有价值的事情，更开源、更开放。
